packages/samples/vuejs/src/main.ts

```diff
@@ -1,4 +1,24 @@
 import {createApp} from 'vue';
 import App from './App.vue';
+import * as components from './components/index.ts';
```

Comment:
this gives me an error `An import path cannot end with a '.ts' extension. Consider importing './components/index.js' instead.` and if I remove the `.ts` it complains about the imported components, 
```
Element implicitly has an 'any' type because expression of type 'string' can't be used to index type 'typeof import("/Users/liviarett/coveo/ui-kit/packages/samples/vuejs/src/components/index")'.
  No index signature with a parameter of type 'string' was found on type 'typeof import("/Users/liviarett/coveo/ui-kit/packages/samples/vuejs/src/components/index")'.
```
what should I do?

Reply:
Does this work?
```suggestion
import * as components from './components';
```

Reply:
no, it gives me the same error as using index without `.ts`

Reply:
actually, so does changing it to `.js` 😩